arduino/sensors/TMP102_TempatureSensor/TMP102_TempatureSensor.ino

```diff
@@ -0,0 +1,47 @@
+const int ALERT_PIN = A3;
+#include<Wire.h>
+#include <Sensor2357.h>
+#include <SparkFunTMP102.h>
+TMP102 sensor0;
```

Comment:
let's rename this to a less-vague name. Next year when we train the new students we do not want to confuse them with vague sensor names. Maybe try `tempSensor` just to make sure we know what the sensor is doing.